bin/pcp-ping

```diff
@@ -4,6 +4,7 @@ require 'pcp/client'
 start_time = Time.now.to_f
 
 Thread.new { EM.run }
+Thread.pass until EM.reactor_running?
```

Comment:
To be more critical,  if it fails to start than this example program could hang.  As it's an example program I don't feel the need to try to police eventmachines ability to satisfy its own API.

There's no reasonable way to detect this without picking a magic number and putting ourself on the wrong end of a race.  Should it have started in 1 second?  What about the day it took legitimately 1.5?